### PR TITLE
Added clarifying notes to PGD locking criteria.

### PR DESCRIPTION
**Pull request by djw-m**
## What Changed?

Extra headings and clarifying paras for DDL and DML locks. 

I can't go further as  changes to app messaging are not done.

Addresses:  https://enterprisedb.atlassian.net/browse/BDR-4195

### CODE REVIEW

**Comment by vaijayanti-b** [comment]
Looks good.

**Comment by djw-m** [comment]
> Looks good.

Could you review and approve with that?

File: product_docs/docs/pgd/5/ddl.mdx [hunk 1]
```diff
@@ -177,6 +186,10 @@ can be reached and are keeping up reasonably well with the current write rate.
 If such DDL commands must be performed while a node is down, first remove the
 down node from the configuration.
 
+A global DML lock is granted when *ALL* the eligible nodes, data and witness, agree to grant it.
+Subscriber-only nodes are not eligible to participate.
+
```
Comment (vaijayanti-b) [inline product_docs/docs/pgd/5/ddl.mdx:191]: I looked at the code for this - and also the ticket mentions it - witness nodes do not contribute to DML locks. They participate in DDL and advisory locks but not DML. Therefore this should be:
A global DML lock is granted when *ALL* the eligible data nodes agree to grant it.
Subscriber-only nodes and witness nodes are not eligible to participate.

Comment (vaijayanti-b) [reply product_docs/docs/pgd/5/ddl.mdx:191]: bdr_gl_node_participates_in_locking

**Review by vaijayanti-b — APPROVED** [review]
Looks good.

**Pull request merged**